src/components/currentLesson/_currentLesson.scss

```diff
@@ -0,0 +1,77 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@500&display=swap');
+
+$font-nunito: 'Nunito', sans-serif;
+@mixin nunito-medium {
+  font-family: $font-nunito;
+  font-weight: 500;
+}
+
+.cl-card {
+  max-width: 300px;
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  overflow: hidden;
+  margin: 20px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding: 10px 7px;
+}
+
+.cl-card-image {
+  width: 100%;
+  max-height: 170px;
+  display: block;
+  object-fit: fill;
+  border-radius: 15px;
+}
+
+.cl-play-button {
+  position: absolute;
+  top: 33%;
+  left: 48%;
+  transform: translate(-50%, -50%);
+  width: 30px;
+  height: 30px;
+  border: none;
+  background-color: transparent;
+  font-size: 40px;
+  color: black;
+}
+
+.cl-card-content {
+  padding: 15px;
+  background: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.cl-course-name {
+  font-weight: bold;
+  font-size: 1.2em;
+  margin: 0 0 10px;
+}
+
+.cl-lesson-name {
+  font-size: 1.05em;
+  margin: 0 0 10px;
+}
+
+.cl-card-info {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 0.9em;
+}
+
+.cl-prop-name {
+  display: flex;
+  align-items: center;
+  opacity: 50%;
+}
+
+.cl-prop-value {
+  font-weight: bold;
+  opacity: 50%;
+}
```

Comment:
```suggestion
.cl-card {
  max-width: 300px;
  border-radius: 10px;
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
  overflow: hidden;
  margin: 20px;
  position: relative;
  display: flex;
  flex-direction: column;
  padding: 10px 7px;
  &-image{
     width: 100%;
     max-height: 170px;
     display: block;
     object-fit: fill;
     border-radius: 15px;
  }
  & .cl-play-button {
  position: absolute;
  top: 33%;
  left: 48%;
  transform: translate(-50%, -50%);
  width: 30px;
  height: 30px;
  border: none;
  background-color: transparent;
  font-size: 40px;
  color: black;
  }
  & .cl-card-content {
  padding: 15px;
  background: white;
  display: flex;
  flex-direction: column;
  }

  & .cl-course-name {
  font-weight: bold;
  font-size: 1.2em;
  margin: 0 0 10px;
  }

  & .cl-lesson-name {
  font-size: 1.05em;
  margin: 0 0 10px;
  }

  & .cl-card-info {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  font-size: 0.9em;
  }

  & .cl-prop-name {
  display: flex;
  align-items: center;
  opacity: 50%;
  }

  & .cl-prop-value {
  font-weight: bold;
  opacity: 50%;
  }
}
```
Use attachment structure in css(scss). If somebody create className with the same name in other component - it will not interrupt your styles.

Reply:
fixed